README.md

```diff
@@ -1074,6 +1075,51 @@ crud.len('customers')
 ...
 ```
 
+### Storage info
+
+```lua
+-- Get storages status
+local result, err = crud.storage_info(opts)
+```
+
+where:
+
+* `opts`:
+  * `timeout` (`?number`) -  maximum time (in seconds) to wait for response from
+  cluster instances.
+
+Returns storages status table by instance UUID or nil with error. Status table fields:
+"status" contains a string representing the status:
```

Comment:
I think it's better to write table field names and values as inline code (`status`, `"running"` etc).